charts/probate-frontend/Chart.yaml

```diff
@@ -3,10 +3,10 @@ appVersion: "1.0"
 description: A Helm chart for the HMCTS probate product
 name: probate-frontend
 home: https://github.com/hmcts/probate-frontend
-version: 2.3.15
+version: 2.3.16
 dependencies:
   - name: nodejs
-    version: 2.4.4
+    version: 2.4.5
```

Comment:
Please upgrade to latest chart version of nodejs -> https://github.com/hmcts/chart-nodejs/releases